Lib/profiling/tracing/__init__.py

```diff
@@ -0,0 +1,207 @@
+"""Tracing profiler for Python.
+
+This module provides deterministic profiling of Python programs by tracing
+every function call and return. It's based on the cProfile implementation.
+"""
+
+__all__ = ["run", "runctx", "Profile"]
+
+import _lsprof
+import importlib.machinery
+import importlib.util
+import io
+import profile as _pyprofile
```

Comment:
Will this trigger the (new) deprecation warning for `profile`? Should the `_Utils` class / docstrings be moved to some common location?

Reply:
Yeah I think I am going to move this to a different location and import it from the other place